### DISCUSSION TITLE
How can I change mitmweb style?

### DISCUSSION THREAD

**Discussion by Stephen0910** [mitmproxy/mitmproxy#7274] [Q&A]
1. How can I change "comment" style in mitmweb, such as put the value in the center or color it red?
![image](https://github.com/user-attachments/assets/112b8889-a15e-4c94-aa3b-f0c3c573c184)

2. May I add a new web column in mitmweb? So I can easily notice the error response.

**Comment by Stephen0910** [comment]
@mhils

**Comment by mhils** [comment]
1. Tagging me explicitly does not generate a faster response.
2. Opening a new issue does not help either.
3. That functionality does not exist at the moment. You can add markers if you want something that pops out more.

**Reply by Stephen0910** [reply]
I am very sorry for opening a new issue. It just seems that there are relatively few people coming to the discussion area. Thanks. I have confirmed that there is currently no existing function implementation. I can think of other ways.
